mrp_restrict_lot/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "MRP Restrict Lot",
+    "version": "14.0.1.0.0",
+    "category": "Manufacturing",
+    "author": "Akretion,Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/manufacture",
+    "license": "AGPL-3",
+    "depends": [
+        "stock_restrict_lot",
+        "mrp",
+    ],
+    "installable": True,
+}
```

Comment:
missing keys : maintainers, development_status

Reply:
not mandatory I guess, but I've added it anyway, thanks